sfputil/main.py

```diff
@@ -723,7 +749,10 @@ def fetch_error_status_from_platform_api(port):
         physical_port_list = logical_port_name_to_physical_port_list(logical_port_name)
         port_name = get_physical_port_name(logical_port_name, 1, False)
 
-        output.append([port_name, output_dict.get(physical_port_list[0])])
+        if is_rj45_port_from_api(logical_port_name):
```

Comment:
How about sfp.get_error_description() for RJ45 just return 'N/A' so that we don't need do anything here?